test/onnx/test_onnxscript_runtime.py

```diff
@@ -0,0 +1,132 @@
+# Owner(s): ["module: onnx"]
+
+"""Test the support on onnxscript in PyTorch-ONNX converter with onnxruntime."""
+from typing import List
+
+import onnx_test_common
+import onnxscript
+import torch
+from onnxscript.onnx_types import FLOAT
+from torch.onnx._internal import jit_utils
+from torch.testing._internal import common_utils
+
+
+class TestONNXScriptRuntime(onnx_test_common._TestONNXRuntime):
+
+    # opset version is
+    # 1. local function is supported after opset 15
+    # 2. onnx-script requires users to determine opset in local function
+    opset_version = 15
+
+    def test_selu_from_onnxscript_example(self):
+
+        x = torch.randn(1, 2, 3, 4, requires_grad=True)
+        model = torch.nn.SELU()
+
+        from onnxscript.onnx_opset import opset15 as op
```

Comment:
Coming from https://github.com/pytorch/pytorch/pull/86906/files#r1013044361

I see this `opset15` needs to match or be compatible with the `opset_version` down in 
```python
torch.onnx.register_custom_op_symbolic(
            symbolic_name="aten::selu",
            symbolic_fn=custom_selu,
            opset_version=self.opset_version,
        )
```

Then if `opset_version` is compatible with export opset version, the onnx script function gets invoked.

So there are 3 versions. I think going forward we can try merge the first two. Read and use the opset version from the onnx_fn when registering custom op using onnx_script. We might even want to require exact opset version matching for onnxscript custom op to reduce likelihood of version issue (I'm not sure maybe @justinchuby already made the change enforcing it for all custom ops).

Reply:
Currently all registered ops customs or not will follow this version resolution logic:

https://github.com/pytorch/pytorch/blob/893f8e3790df47b165025c9e6b5b37b85bdfd501/torch/onnx/_internal/registration.py#L22-L24

We can update it to create different behaviors if desired

Reply:
@justinchuby Let's have an offline meeting about this!

Reply:
One way of doing this, not elegant though, is to hide the symbolic_fn and aten op information inside onnx-script function, for example, in script decorator. In such case, we don't have to call another `register_custom_op_symbolic` for aten op, and we only need user to finish onnx-script (which makes sense they specify what aten op they would like to replace with this onnx-script function.). However, we need to find a way to parse and understand what op version we use inside the onnx function.

```python
    @onnxscript.script(custom_opset, symbolic_fn=custom_selu, aten_op="aten::selu")  # TODO: Expose used opset version 
    def Selu(X):
        alpha = 1.6732632423543772848170429916717
        gamma = 1.0507009873554804934193349852946
        alphaX = op.CastLike(alpha, X)
        gammaX = op.CastLike(gamma, X)
        neg = gammaX * (alphaX * op.Exp(X) - alphaX)
        pos = gammaX * X
        zero = op.CastLike(0, X)
        return op.Where(X <= zero, neg, pos)

    def custom_selu(g, X):
        # domain of the Op should be aligned with onnx-script
        # setType API is required for custom Op to support
        # torchscript shape type inference
        print("custom_selu is used!")
        return g.onnxscript_op(Selu, X).setType(X.type())

  #  torch.onnx.register_custom_op_symbolic(
  #      symbolic_name="aten::selu",
  #      symbolic_fn=custom_selu,
  #      opset_version=15,
  #  )
```

Reply:
In offline meeting with @justinchuby, we decided to go with this version for now, as the integration of symbolic_fn and onnx-script is in progress, we can just wait for the new design for the next release.